backend/services/mgmt/v1alpha1/integration_tests/user-account-service_integration_test.go

```diff
@@ -525,7 +525,7 @@ func (s *IntegrationTestSuite) Test_UserAccountService_IsAccountStatusValid_Neos
 	requireNoErrResp(s.T(), resp, err)
 
 	require.False(s.T(), resp.Msg.GetIsValid())
-	require.NotEmpty(s.T(), resp.Msg.GetReason())
+	require.NotEmpty(s.T(), resp.Msg.GetAccountStatus())
```

Comment:
Does this need to change?
If anything I would expect this to be unchanged, but new lines added for the expected state of the new additions (used, allowed count and the account status)

Reply:
updated tests